docs/contributing/index.rst

```diff
@@ -40,52 +40,82 @@ Preperations for a new Python version
+      As we try to be **as explicit as possible**, all language features should have a corresponding ``visit_<AST Node>``.
+      That follows the Zen of Python:
 
-Additionally modify ``.meta.toml`` and run the ``meta/config`` script to update the following files:
+      .. code-block:: python
+          :emphasize-lines: 5
 
-* ``/setup.py`` - Check that the new Python version classifier has been added ``"Programming Language :: Python :: <version>",``, and that the ``python_requires`` section has been updated correctly.
-* ``/tox.ini`` - Check that a ``testenv`` entry is added to the general ``envlist`` statement.
-* ``/.github/workflows/tests.yml`` - Check that a corresponding Python version entry has been added to the matrix definition.
+          >>> import this
+          The Zen of Python, by Tim Peters
 
-Run the tests via ``tox``.
+          Beautiful is better than ugly.
+          Explicit is better than implicit.
+          Simple is better than complex.
+          Complex is better than complicated.
+          Flat is better than nested.
+          Sparse is better than dense.
+          Readability counts.
+          Special cases aren't special enough to break the rules.
+          Although practicality beats purity.
+          Errors should never pass silently.
+          Unless explicitly silenced.
+          In the face of ambiguity, refuse the temptation to guess.
+          There should be one-- and preferably only one --obvious way to do it.
+          Although that way may not be obvious at first unless you're Dutch.
+          Now is better than never.
+          Although never is often better than *right* now.
+          If the implementation is hard to explain, it's a bad idea.
+          If the implementation is easy to explain, it may be a good idea.
+          Namespaces are one honking great idea -- let's do more of those!
 
-Create a pull request.
+  * Add a corresponding changelog entry.
+  * Additionally modify ``.meta.toml`` and run the ``meta/config`` script (for details see: https://github.com/mgedmin/check-python-versions) to update the following files:
+
+    * ``/setup.py`` - Check that the new Python version classifier has been added ``"Programming Language :: Python :: <version>",``, and that the ``python_requires`` section has been updated correctly.
+    * ``/tox.ini`` - Check that a ``testenv`` entry is added to the general ``envlist`` statement.
+    * ``/.github/workflows/tests.yml`` - Check that a corresponding Python version entry has been added to the matrix definition.
+    * ``/docs/conf.py`` - Add Python version to ``intersphinx_mapping`` list
+
+  * Run the tests via ``tox`` locally before push.
 
 Enable a Python Feature in RestrictedPython
 +++++++++++++++++++++++++++++++++++++++++++
 
 To enable a certain functionality in RestrictedPython, do the following:
 
-#. Add a feature request on GitHub.
+#. Add a feature request on GitHub to discuss.
```

Comment:
It might not be obvious that one needs to create an issue to make a feature request. Also note that the link to create a new issue has only two options:

- Report a bug
- Report a security issue

...neither of which is a feature request. 😱 I'd suggest adding that as an option to the `.github` files, either for the [organization](https://github.com/zopefoundation/.github/blob/master/.github/ISSUE_TEMPLATE/) or the local repo. I am not sure where it is configured for this repo.

```suggestion
#. [Create a new issue on GitHub](https://github.com/zopefoundation/RestrictedPython/issues/new/choose), requesting the new feature, for discussion.
```